app/views/api/v3/analytics/user_analytics/show_v1.html.erb

```diff
@@ -50,10 +50,10 @@
           </h1>
           <div class="mb-24px">
             <p class="m-0px mb-8px p-0px pl-16px ta-left fw-normal fs-16px c-black">
-              <span class="fw-bold"><%= number_with_delimiter(@total_follow_ups) %></span> <%= t("progress_tab.achievements_card.follow_up_visits_section_title", visit_or_visits: "visit".pluralize(@total_follow_ups)) %>
+              <span class="fw-bold"><%= number_with_delimiter(@service.total_follow_ups) %></span> <%= t("progress_tab.achievements_card.follow_up_visits_section_title", visit_or_visits: "visit".pluralize(@service.total_follow_ups)) %>
```

Comment:
> Shall we change @total_follow_ups and @total_registrations (lines 21-22 in user_analytics_controller.rb) to use @service instead of MonthlyProgressComponent?

Let's keep this as it is because we have to delete these variables `@total_follow_ups` and `@total_registrations` as it is used in the old progress tab code. I will add this to the cleanup story so that we don't miss it.

Reply:
Let's leave a comment in the controller as a reminder.